examples/aws/ingress/README.md

```diff
@@ -0,0 +1,22 @@
+# Overview 
+
+This document describes the steps on how to configure the GraphDB helm chart to use ingress on AWS EKS
+
+## Prerequisites
+
+* EKS Cluster: Ensure you have an EKS cluster up and running.
+* [Installing AWS Load Balancer Controller](https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.8/deploy/installation/)
+* [Ingress Setup](https://docs.aws.amazon.com/eks/latest/userguide/alb-ingress.html)
+* [Exposing kubernetes applications via ingress](https://aws.amazon.com/blogs/containers/exposing-kubernetes-applications-part-1-service-and-ingress-resources/)
+
+## Example
```

Comment:
Always leave an empty line around headings and paragraphs.

Reply:
There is one more that should be addressed but another time.

Reply:
Fixed